frontend/src/components/VSearchResultsGrid/VImageGrid.vue

```diff
@@ -1,7 +1,7 @@
 <template>
   <section>
     <VGridSkeleton
-      v-if="!images.length && !fetchState.isFinished"
+      v-if="images != null && !images.length && !fetchState.isFinished"
```

Comment:
```suggestion
      v-if="!images?.length && !fetchState.isFinished"
```
Optional chaining operator `?`[^1] can make this expression cleaner. It returns `undefined` if `images` is null or undefined, and does not try to access the `.length` property.

[^1]: https://javascript.info/optional-chaining